edison/src/main/java/edu/illinois/cs/cogcomp/edison/annotators/GazetteerViewGenerator.java

```diff
@@ -97,11 +99,13 @@
 
     private Set<String> ignore;
 
-    public GazetteerViewGenerator(String directory, String viewName) throws Exception {
-        this(directory, true, viewName);
+    private boolean loadFromFromClasspath = false;
+
+    public GazetteerViewGenerator(String directory, String viewName, boolean loadFromFromClasspath) throws Exception {
+        this(directory, true, viewName, loadFromFromClasspath);
     }
 
-    public GazetteerViewGenerator(String directory, boolean gzip, String viewName) throws Exception {
+    public GazetteerViewGenerator(String directory, boolean gzip, String viewName, boolean loadFromFromClasspath) throws Exception {
```

Comment:
The `loadFromFromClasspath` option will not work right? Because the dependencies have been commented out. Is the idea to allow loading from classpath if the user adds the resources explicitly in the pom file?

Also a typo in the variable name.